.github/workflows/mn-macos-snapshot.yml

```
@@ -11,36 +11,36 @@ jobs:
     name: Builds OS X Native CLI
     runs-on: macos-latest
     steps:
-      - name: Checkout the repository
+      - name: "⬇ Checkout the repository"
         uses: actions/checkout@v3
-      - name: Setup GraalVM CE
-        uses: graalvm/setup-graalvm@v1.0.12
         with:
-          version: '22.3.1'
+          fetch-depth: 0
+      - name: "☕️ Setup GraalVM CE"
+        uses: graalvm/setup-graalvm@v1
```

```suggestion
        uses: graalvm/setup-graalvm@v1.1.4.2
```

see: [v1.1.4.2](https://github.com/graalvm/setup-graalvm/releases/tag/v1.1.4.2)